### PR TITLE
be clear on case-sensitive

### PR DESCRIPTION
**Pull request by haitch**
also instruct to check extension status form azure portal

### CODE REVIEW

File: articles/log-analytics/log-analytics-azure-vm-extension.md [hunk 1]
```diff
@@ -58,6 +58,12 @@ You can install the agent for Log Analytics and connect the Azure virtual machin
    ![Connected](./media/log-analytics-azure-vm-extension/oms-connect-azure-05.png)
 
 ## Enable the VM extension using PowerShell
+When you configure your virtual machine by using PowerShell, you need to provide the **workspaceId** and **workspaceKey**, the property name in your json configuration is **case-sensitive**.
```
Comment (snarkywolverine) [inline articles/log-analytics/log-analytics-azure-vm-extension.md]: I'd make these two sentences - ...**workspaceId** and **workspaceKey**.  Note that the property names in your json configuration are **case-sensitive**.

File: articles/log-analytics/log-analytics-azure-vm-extension.md [hunk 2]
```diff
@@ -357,7 +360,19 @@ You can deploy a template by using the following PowerShell command:
 New-AzureRmResourceGroupDeployment -ResourceGroupName $resourceGroupName -TemplateFile $templateFilePath
 ```
 
-## Troubleshooting Windows Virtual Machines
+## Troubleshoot when VM are not reporting to Log Analytics
+Usually you should get notified with a message when thing doesn't work, at least from Azure portal or Azure powershell.
```
Comment (snarkywolverine) [inline articles/log-analytics/log-analytics-azure-vm-extension.md]: Usually you will receive a message when things don't work, from either Azure portal or Azure powershell.

File: articles/log-analytics/log-analytics-azure-vm-extension.md [hunk 3]
```diff
@@ -357,7 +360,19 @@ You can deploy a template by using the following PowerShell command:
...
-## Troubleshooting Windows Virtual Machines
+## Troubleshoot when VM are not reporting to Log Analytics
+Usually you should get notified with a message when thing doesn't work, at least from Azure portal or Azure powershell. 
+
+1. Sign into the [Azure portal](http://portal.azure.com).
+2. Find the VM and open VM details.
+3. Click on Extensions to check if OMS extension is enabled or not.
+
+   ![VM Extension View](./media/log-analytics-azure-vm-extension/oms-vmview-extensions.png)
+4. Click on the MicrosoftMonitoringAgent or OmsAgentForLinux extension and view details.
```
Comment (snarkywolverine) [inline articles/log-analytics/log-analytics-azure-vm-extension.md]: Probably should include reference to MicrosoftMonitoringAgent (Windows) or OmsAgentForLinux (Linux) for clarity

File: articles/log-analytics/log-analytics-azure-vm-extension.md [hunk 4]
```diff
@@ -357,7 +360,19 @@ You can deploy a template by using the following PowerShell command:
 New-AzureRmResourceGroupDeployment -ResourceGroupName $resourceGroupName -TemplateFile $templateFilePath
 ```
 
-## Troubleshooting Windows Virtual Machines
+## Troubleshoot when VM are not reporting to Log Analytics
```
Comment (richrundmsft) [inline articles/log-analytics/log-analytics-azure-vm-extension.md]: ## Troubleshooting the Log Analytics VM extension

File: articles/log-analytics/log-analytics-azure-vm-extension.md [hunk 5]
```diff
@@ -357,7 +360,19 @@ You can deploy a template by using the following PowerShell command:
...
+Usually you should get notified with a message when thing doesn't work, at least from Azure portal or Azure powershell. 
+
+1. Sign into the [Azure portal](http://portal.azure.com).
+2. Find the VM and open VM details.
+3. Click on Extensions to check if OMS extension is enabled or not.
```
Comment (richrundmsft) [inline articles/log-analytics/log-analytics-azure-vm-extension.md]: Wrap the name of the thing to click in  \*\* to make it bold in the instructions, e.g. **Extensions**

Wrap the name of the extension in \* to make it italics, e.g. *MicrosoftMonitoringAgent* (Windows)

**Pull request merged**